x-pack/solutions/security/plugins/timelines/common/search_strategy/index_fields/index.ts

```diff
@@ -50,14 +50,14 @@ export interface IndexFieldsStrategyRequestById extends IEsSearchRequest {
 }
 
 /**
- * @deprecated use kibana data views api https://github.com/elastic/kibana/blob/83f1fb4f26219f32cab43706db78d544c7bc2f6d/src/plugins/data_views/common/data_views/data_views.ts#L294
+ * @deprecated use kibana data views api https://github.com/elastic/kibana/blob/83f1fb4f26219f32cab43706db78d544c7bc2f6d/src/platform/plugins/shared/data_views/common/data_views/data_views.ts#L294
```

Comment:
this path (and the one a few lines below) will never exist. I realize that the person who added the path in the first place pointed to a specific commit instead of main, which I believe was a mistake.

It's not the end of the world, but we should fix this at some point...